### PR TITLE
Fix RNG startup and add Movidius udev rules

### PR DESCRIPTION
**Pull request by zehortigoza**

### CODE REVIEW

File: recipes-support/movidius/movidius_0.1bb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+SUMMARY = "Movidius udev rules"
```
Comment (lucasdemarchi) [inline recipes-support/movidius/movidius_0.1bb]: wrong file extension

Comment (lucasdemarchi) [inline recipes-support/movidius/movidius_0.1bb]: and you also need to explicitly add this to the intel-aero-image

Comment (zehortigoza) [inline recipes-support/movidius/movidius_0.1bb]: oh I fixed this and amend to the wrong commit, fixing...

Comment (zehortigoza) [inline recipes-support/movidius/movidius_0.1bb]: > and you also need to explicitly add this to the intel-aero-image

I did it on the next commit but if you want we can squash

File: recipes-support/rng-tools/rng-tools_5.bbappend [hunk 2]
```diff
@@ -1 +1,9 @@
-INITSCRIPT_PARAMS = "start 08 2 3 4 5 . stop 30 0 6 1 ."
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI += "file://rngd.service"
+
+do_install_append() {
+        install -d "${D}${sysconfdir}/systemd/system"
+        install -m 0644 ${WORKDIR}/rngd.service ${D}${sysconfdir}/systemd/system/rngd.service
+}
+
+SYSTEMD_SERVICE_${PN} = "rngd.service"
```
Comment (zehortigoza) [inline recipes-support/rng-tools/rng-tools_5.bbappend]: This line did not automatically started the service, looking why... please do not merge yet

Comment (lucasdemarchi) [inline recipes-support/rng-tools/rng-tools_5.bbappend]: It does auto enable if the recipe inherits systemd's class

**Comment by zehortigoza** [comment]
now working after add "inherits systemd"

File: recipes-support/movidius/files/LICENSE [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+The above copyright notice and this permission notice shall be included in 
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR 
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, 
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE 
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER 
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN 
+THE SOFTWARE.
```
Comment (magicrub) [inline recipes-support/movidius/files/LICENSE]: Woah... Shouldn't this license mention the license type And Intel?

Comment (lucasdemarchi) [inline recipes-support/movidius/files/LICENSE]: this is for these udev rules only, not anything else. I would even remove it, but doesn't really matter

File: recipes-support/rng-tools/files/rngd.service [hunk 4]
```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Hardware RNG Entropy Gatherer Daemon
+
+[Service]
+#EnvironmentFile=/etc/conf.d/rngd
```
Comment (lucasdemarchi) [inline recipes-support/rng-tools/files/rngd.service]: this line should be removed rather than commented

Comment (zehortigoza) [inline recipes-support/rng-tools/files/rngd.service]: Done

**Pull request merged**